src/javascript/app/pages/trade/purchase.js

```diff
@@ -105,6 +105,11 @@ const Purchase = (() => {
                     BinarySocket.wait('get_account_status').then(response => {
                         confirmation_error.setVisibility(1);
                         let message = error.message;
+                        if (/CompanyWideLimitExceeded/i.test(error.code)) {
+                            const redirect = '<a href="https://www.binary.com/en/terms-and-conditions.html?anchor=trading-limits#legal-binary" target="_blank" rel="noopener noreferrer">';
+                            const redirect_close = '</a>';
+                            message = localize('No further trading is allowed on this contract type for the current trading session. For more info, refer to our [_1]terms and conditions[_2].',[redirect, redirect_close]);
+                        }
                         if (/NoMFProfessionalClient/.test(error.code)) {
```

Comment:
```suggestion
                        else if (/NoMFProfessionalClient/.test(error.code)) {
```
Error code can match only one code. So, it's better to add `else if`.

Reply:
alrightss